jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ServerTemplateSelectorMenuBuilderTest.java

```diff
@@ -248,4 +249,100 @@ public void testServerTemplateChangeHandler(){
         inOrder.verify(viewWidget).updateSelectedValue(serverTemplateId);
         inOrder.verify(serverTemplateSelectedEvent).fire(any());
     }
-}
\ No newline at end of file
+
+    @Test
+    public void testLoadServerTemplatesForSelectedServerTemplateIsNull(){
+        final String serverTemplateId1 = "id1";
+        final ServerTemplate st1 = new ServerTemplate(serverTemplateId1,
```

Comment:
@bxf12315 This is not corrected by 'Reformat code' following our code conventions but we agreed to keep the parameters at the same line if it's possible. Maybe you can follow the same idea in this whole test. The point is  incrementally update the code we need to change for other reasons, avoiding to do a big update just for reformat the code.